launch_testing/launch_testing/__init__.py

```diff
@@ -35,6 +35,7 @@ def __init__(
         self.name = name
         self.launch_descriptor = launch_descriptor
         self.expected_lines = expected_lines
+        self.expected_output = b'\n'.join(self.expected_lines)
```

Comment:
If launch was documented better, I would know how to use the existing handlers in order to implement what I need for case of matching a multi-line output expectation.